utils/hash/hash.go

```diff
@@ -0,0 +1,34 @@
+package hash
+
+import (
+	"encoding/binary"
+	"encoding/json"
+	"fmt"
+	"hash/fnv"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+)
+
+// ComputePodTemplateHash returns a hash value calculated from the Rollout's pod template.
```

Comment:
nit: "from the Rollout's pod template." -> ""from the pod template as the podtemplate could be from an experiment

Reply:
fixed